data/human/culture conversations.txt

```diff
@@ -120,6 +149,38 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Forest Fire"
+	minor
+	source
+		government "Free Worlds"
+		attributes "frontier"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning smoke permeates the air as you step off your craft, causing you to rush towards the nearest shelter, coughing along the way. Once there, you find people shouting, rushing about to get tape plastered onto the windows. No one seems to have noticed your arrival and none give an explanation.`
+			choice
+				`	"Hey! What's happening? I can help!" (Shout.)`
+					goto help
+				`	(Look for someone not in motion.)`
+			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll blow over us."`
```

Comment:
"blow over us" doesn't seem like a natural way to describe a fire.

Reply:
This might be an Australian phrase. If anyone has a more broadly used alternative I'll accept that.

Reply:
Hoping something "blows over" as in hoping it stops being an issue is definitely a phrase people use in the US. The issue is that, especially because you mentioned the wind previously, it sounds like they're hoping the wind blows the fire over them, which isn't really how fires work. If that was the intent, this might work?
```suggestion
			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping this blow over."`
```

Reply:
But... it is how fire works? Like if you hunker down in a secure (non-flammable) shelter in the face of a firefront being driven by wind, the front literally blows over you.

Reply:
I guess my thought was that fire doesn't really skip an area. Maybe it would burn around the shelter without harming it, as it is fire-proof, but it doesn't seem like it would just blow over an area without touching it.
However, this is purely based on intuition and is based on no actual knowledge of fire. Feel free to ignore this suggestion.

Reply:
> but it doesn't seem like it would just blow over an area without touching it.

Yeah you'd be surprised, I think you'd be fascinated by what does and doesn't get burnt in big wind-driven fires.

Reply:
```suggestion
			`	You seek out a pregnant lady sitting on a chair, a mask worn as she explains: "A large fire has sprung up nearby, and the wind is fanning it towards the spaceport. Might have been some fool playing with fire, or some pirate sending a message to a local. We don't know, could be anything around here really. We're hunkering down and hoping it'll burn out before it does too much damage."`
```
How about this? I do generally agree with Arrow on "blowing over us" being an odd way to describe this fire.

Reply:
Let's go politically neutral and just take the change.